src/main/resources/db/changelog/changes/schema.xml

```diff
@@ -376,6 +376,7 @@
     </changeSet>
 
     <changeSet id="add_level" author="Darina">
+        <validCheckSum>8:0b8ffa56b61fb87ce6cf127b376a30f2</validCheckSum>
```

Comment:
Not a valid changeset passing.
You need to revert a created changeset and create a new one

Reply:
omg, ok😰